src/freqresp.jl

```diff
@@ -19,6 +19,66 @@ of system `sys` over the frequency vector `w`."""
     [evalfr(sys[i,j], s)[] for s in s_vec, i in 1:ny, j in 1:nu]
 end
 
+# TODO Most of this logic should be moved to the respective options, e.g. bode
+# TODO Less copying of code
+@autovec () function freqresp(sys::LTISystem, lims::Tuple; style=:none)
```

Comment:
You return a tuple later on, so depending on which of the values should be `vec`'d you need to give the indices for those values in the tuple. No indices is if the function returns no tuple, but the whole object that should be `vec`'d.

Reply:
```suggestion
@autovec (1) function freqresp(sys::LTISystem, lims::Tuple; style=:none)
```

Would then be the way assuming the grid should not be changed.

Reply:
Thanks. This is what happens when you copy code and forget to look at it again..